src/test/kotlin/graphics/scenery/tests/examples/volumes/ServerApplication.kt

```diff
@@ -0,0 +1,137 @@
+        volume.name = "volume"
+        volume.colormap = Colormap.get("viridis")
+        volume.spatial {
+            position = Vector3f(0.0f, 0.0f, -3.5f)
+            rotation = rotation.rotateXYZ(0.05f, 0.05f, 0.05f)
+            scale = Vector3f(20.0f, 20.0f, 20.0f)
+        }
+
+        volume.transferFunction = TransferFunction.ramp(0.1f, 0.5f)
+        scene.addChild(volume)
+
+
+        //Step 2: create a volume manager for vdi and add the volume to it:
+        val vdiVolumeManager = VDIVolumeManager(hub, windowWidth, windowHeight, maxSupersegments, scene).createVDIVolumeManger()
+        vdiVolumeManager.add(volume)
+
+        //Step 3:  save the standard volume manger, the one who was first created with the volume
+        val standardVolumeManager: VolumeManager = hub.get<VolumeManager>() as VolumeManager
+
+        thread {
+            while (true) {
+                volume.spatial {
+                    rotation = rotation.rotateY(0.003f)
+                }
+                Thread.sleep(5)
+            }
+        }
+
+        //Step 4: render volume first time
+        renderVolume(volume)
+
+        //Step 5: switch between different volume managers
+        var currentlyVolumeRendering = true
+        var firstVDI = true
+
+        thread {
+                while (true) {
+                    val switchMode = scene.find("EmptyNode") as? EmptyNode
+
+                    if (switchMode != null) {
+
+                        if (!currentlyVolumeRendering && switchMode.value.equals("toVR")) {
+
+                            logger.warn("Volume Rendering")
+
+                            renderer?.vdiStreaming = false
+
+                            standardVolumeManager.replace(standardVolumeManager)
+                            renderVolume(volume)
+
+                            currentlyVolumeRendering = !currentlyVolumeRendering
+
+                        }
+                        else if (currentlyVolumeRendering && switchMode.value.equals("toVDI")) {
+
+                            logger.warn("VDI Streaming")
+
+                            renderer?.recordMovie()
+                            vdiVolumeManager.replace(vdiVolumeManager)
+
+//                            renderer?.vdiStreaming = true
+//
+//                            if (firstVDI) {
+//                                val volumeDimensions3i = Vector3f(volume.getDimensions().x.toFloat(), volume.getDimensions().y.toFloat(), volume.getDimensions().z.toFloat())
+//                                val model = volume.spatial().world
+//                                renderer?.streamVDI("tcp://0.0.0.0:6655", cam, volumeDimensions3i, model, context)
+//                                firstVDI = false
+//                            }
+
+                            currentlyVolumeRendering = !currentlyVolumeRendering
+                        }
+                    }
+
+                }
+            }
```

Comment:
Please fix indentation, unnecessary comments, line spaces etc. here and alsewhere